shell/platform/linux/fl_view.cc

```diff
@@ -43,21 +49,22 @@ struct _FlView {
   FlMouseCursorPlugin* mouse_cursor_plugin;
   FlPlatformPlugin* platform_plugin;
   FlTextInputPlugin* text_input_plugin;
+  FlPlatformViewsPlugin* platform_views_plugin;
+
+  FlGestureHelper* gesture_helper;
 
   GList* gl_area_list;
   GList* used_area_list;
 
+  std::map<GtkWidget*, FlClippingView*>* current_clipping_views;
```

Comment:
This should be `GHashTable` for consistency with the other code.

Reply:
Done.

Reply:
We cannot use C++ objects in GObject directly. Because constructors of C++ objects should be called after memory allocation. But GObjects are allocated by malloc. Though we can use in-place new and direct call to destructors, that is not suggested.